jbpm-wb-common/jbpm-wb-common-api/src/main/java/org/jbpm/workbench/common/model/GenericErrorSummary.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.common.model;
+
+public class GenericErrorSummary extends GenericSummary<Long>{
```

Comment:
Please make this class abstract, also receiving a generic id type as in the _GenericSummary_.
Other important fixes in here should include, adding equals, hashcode and toString methods.